src/System.Drawing.Common/src/System/Drawing/CharacterRange.cs

```diff
@@ -14,7 +16,7 @@ public struct CharacterRange
         /// <summary>
         /// Initializes a new instance of the <see cref='CharacterRange'/> class with the specified coordinates.
         /// </summary>
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")]
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")]
```

Comment:
@stephentoub I assume we will never enable CA1704? In which case we can delete all the suppressions

Reply:
Yes.  I've been waiting to open an issue about it until we've completed fixing all the issues associated with all the rules we plan to enable, but we're pretty close to that.  At that point, we should go through the code and delete all nop suppressions.